### PR TITLE
Adding EPEL-Next task for Centos 9 Stream

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

File: roles/epel_repositories/defaults/main.yml [hunk 1]
```diff
@@ -2,3 +2,4 @@
 epel_repositories_state: present
 epel_repositories_url: https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm
 epel_repositories_name: "{{ epel_repositories_url if (epel_repositories_state == 'present' and ansible_distribution == 'RedHat') else 'epel-release' }}"
```
[the remark this answers is not in the document]
Comment (Odilhao) [reply roles/epel_repositories/defaults/main.yml]: Had to add `centos-stream-repos` and ensure that it's updated.

File: roles/epel_repositories/tasks/main.yml [hunk 2]
```diff
@@ -1,4 +1,12 @@
 ---
+- name: 'Setup Extra Common if Stream'
+  yum:
+    name: "centos-stream-repos"
+    state: latest
+  tags:
+    - packages
+  when: ansible_distribution_major_version == '9'
```
Comment (evgeni) [inline roles/epel_repositories/tasks/main.yml:8]: This needs an "os == centos" as otherwise that wont exist

Comment (Odilhao) [reply roles/epel_repositories/tasks/main.yml]: Adding `ansible_distribution == 'CentOS'`

**Review by evgeni — APPROVED** [review]

**Comment by Odilhao** [comment]
Err, I don't have merge permission xD

**Pull request merged**